shared-bindings/_stage/__init__.c

```diff
@@ -61,6 +61,7 @@
 //|     :param list layers: A list of the :py:class:`~_stage.Layer` objects.
 //|     :param bytearray buffer: A buffer to use for rendering.
 //|     :param ~displayio.Display display: The display to use.
+//|     :param int: How much should the display be scaled up.
```

Comment:
`:param int scale:`  and maybe say it's the pixels that are scaled.

Reply:
But it's not in pixels, it's in time. 2 makes it 2 times larger.

Reply:
I was thinking if `scale` is 1, one pixel on the display represents one pixel in the buffer. If `scale` is 2, 2x2 pixels on the display represent one pixel in the buffer. It depends on which pixels you mean. We are making different assumptions. I wasn't sure what scaling the "display" meant, but it means scaling the buffer onto the physical display. If you can think of a clearer way to say this, fine, but no real problem if you just leave it this way.

The only other thing is that the word `scale` is missing after `int` in line 64.

Reply:
That's pretty much why I didn't write it — I can't think of a concise and non-confusing way of putting it. For example "with scale=2 every logical pixel corresponds to four physical pixels" creates more questions than it answers: why four? what is a logical pixel?

I reworded it and fixed the missing argument name, but I think I don't want to spend too much time on this, as this is an internal interface, used only by the Python library that wraps it. I will make sure to explain that functionality properly in the library itself.

Reply:
For better English (if you keep the original language):
`How much the display should be scaled up`